src/eth_provider/database/mod.rs

```diff
@@ -61,3 +61,23 @@ impl Database {
         Ok(count)
     }
 }
+
+impl Deref for Database {
+    type Target = MongoDatabase;
+
+    fn deref(&self) -> &Self::Target {
+        &self.0
+    }
+}
+
+impl DerefMut for Database {
+    fn deref_mut(&mut self) -> &mut Self::Target {
+        &mut self.0
+    }
+}
```

Comment:
Should we do this that way or do we decide that we never use `deref` on wrappers? If so I will delete this and do it with another method.

Reply:
Just removed that here.